src/components/Facility/FacilityCover.tsx

```diff
@@ -0,0 +1,75 @@
+import { navigate } from "raviger";
+import { useEffect } from "react";
+
+import useAuthUser from "@/hooks/useAuthUser";
+
+import * as Notification from "@/Utils/Notifications";
+
+import { Cancel, Submit } from "../Common/ButtonV2";
+import Page from "../Common/Page";
+import { CoverImage } from "./CoverImage";
+
+interface FacilityProps {
+  facilityId?: string;
+}
+
+export const FacilityCover = (props: FacilityProps) => {
+  const { facilityId } = props;
+
+  const authUser = useAuthUser();
+  useEffect(() => {
+    if (
+      authUser &&
+      authUser.user_type !== "StateAdmin" &&
+      authUser.user_type !== "DistrictAdmin" &&
+      authUser.user_type !== "DistrictLabAdmin"
+    ) {
+      navigate("/facility");
+      Notification.Error({
+        msg: "You don't have permission to perform this action. Contact the admin",
+      });
+    }
+  }, [authUser]);
+
+  return (
+    <Page
+      title="Cover Image"
+      className="h-[90vh]"
+      crumbsReplacements={{
+        [facilityId || "????"]: { name: "Upload" },
```

Comment:
show the facility name instead of "Upload"

```suggestion
        [facilityId]: { name: "Upload" },
```